identity/webapp/pages/_app.tsx

```diff
@@ -27,5 +33,15 @@ IdentityApp.getInitialProps = async (appContext: AppContext) => {
   // TODO don't store things like this on `ctx.query`
   delete appContext.ctx.query.memoizedPrismic; // We need to remove memoizedPrismic value here otherwise we hit circular object issues with JSON.stringify
 
-  return { ...initialProps, globalContextData };
+  return {
+    ...initialProps,
+    // This is a mega hack as toggles don't work for the identity app as it doesn't use the weird
```

Comment:
Does this warrant having `FIXME:` (or at least a `TODO:`) in front of it for ease of searching through the repo to find stuff we know we have to deal with?